css/style.css

```diff
@@ -1,5 +1,449 @@
-*{
+@font-face {
+  font-family: "GT Walsheim";
+  src: local("GT Walsheim Regular"), local("GTWalsheim-Regular"),
+    url("../fonts/GTWalsheim-regular.woff2") format("woff2");
+  font-weight: normal;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: "GT Walsheim";
+  src: local("GT Walsheim Medium"), local("GTWalsheim-Medium"),
+    url("../fonts/GTWalsheim-medium.woff2") format("woff2");
+  font-weight: 500;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: "GT Walsheim";
+  src: local("GT Walsheim Bold"), local("GTWalsheimBold"),
+    url("../fonts/GTWalsheimBold.woff2") format("woff2");
+  font-weight: bold;
+  font-style: normal;
+  font-display: swap;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "GT Walsheim";
+}
+
+a {
+  text-decoration: none;
+}
+
+.container {
+  width: 95%;
+  max-width: 1200px;
+  margin: auto;
+  overflow: hidden;
+}
+
+.logo {
+  margin-right: 16rem;
+}
+
+.button-small {
```

Comment:
This class should be a modifier class `.button--small`
Please apply BEM @Jefer-Max

Reply:
Done